datastore/datastore.go

```diff
@@ -91,6 +147,26 @@ func init() {
 	boltdb.Register()
 }
 
+// DefaultScopes returns a map of default scopes and it's config for clients to use.
+func DefaultScopes(dataDir string) map[string]*ScopeCfg {
```

Comment:
DefaultScopes generates a default scope config based on the passed data dir. It's just a initialization parameter for scope configs. I'd rather do this than make the caller make more calls.